tests/zkasm/assert.js

```diff
@@ -0,0 +1,68 @@
+const fs = require("fs");
+
+class InstrumentInst {
+    /**
+     * @param {Object} config 
+     * @param {string} [config.outputFile] - the file to which output will be written
+     */
+    constructor(config) {
+        this.config = config;
+        this.results = {};
+    }
+
+    setup() {
+        // `zkevm-proverjs` requires a `setup` function on helper objects.
+    }
+
+    /**
+     * 
+     * @param {Object} ctx - context of zkasm program
+     * @param {Object} tag - information of helper function. 
+     * Helper function must be used in format:
+     * ${assert_eq(regname, regname, tag)}, for example:
+     * ${assert_eq(A, B, test_generated_from_line_3000)}
+     */
+    eval_assert_eq(ctx, tag) {
+        // TODO print verbose error on invalid params
+        // TODO: why arg* is list of two integers with last zero and first value in register???
+        // Is it chunks? If yes they should be merged instead of ignoring second one
+        const arg0 = ctx[tag.params[0].regName][0];
+        const arg1 = ctx[tag.params[1].regName][0];
+        const testname = tag.params[2].varName;
+        this.results[testname] = arg0 === arg1 ? 'pass' : 'fail';
+        return 0;
```

Comment:
Is this value supposed to be written into a register? If yes, why does the return value not depend on the assertion?